ravenpy/models/emulators/canadianshield.py

```diff
@@ -0,0 +1,616 @@
+from collections import defaultdict
+from pathlib import Path
+from typing import cast
+
+import xarray as xr
+from pydantic.dataclasses import dataclass
+
+from ravenpy.config.commands import (
+    HRU,
+    LU,
+    BaseDataCommand,
+    BasinIndexCommand,
+    HRUState,
+    Sub,
+)
+from ravenpy.config.rvs import RVH, RVI
+from ravenpy.models.base import Ostrich, Raven
+
+
+class CANADIANSHIELD(Raven):
+    """
+    The Canadian Shield model is a useful configuration of Raven for
+    Canadian shield basins characterized by shallow soils atop rock,
+    with ample exposed rock and lakes.
+    """
+
+    @dataclass
+    class Params:
+        par_x01: float
+        par_x02: float
+        par_x03: float
+        par_x04: float
+        par_x05: float
+        par_x06: float
+        par_x07: float
+        par_x08: float
+        par_x09: float
+        par_x10: float
+        par_x11: float
+        par_x12: float
+        par_x13: float
+        par_x14: float
+        par_x15: float
+        par_x16: float
+        par_x17: float
+        par_x18: float
+        par_x19: float
+        par_x20: float
+        par_x21: float
+        par_x22: float
+        par_x23: float
+        par_x24: float
+        par_x25: float
+        par_x26: float
+        par_x27: float
+        par_x28: float
+        par_x29: float
+        par_x30: float
+        par_x31: float
+        par_x32: float
+        par_x33: float
+        par_x34: float
+
+    @dataclass
+    class DerivedParams:
+        SUM_X05_X06: float
+        SUM_X16_X17: float
+        POW_X08: float
+        POW_X09: float
+
+    @dataclass
+    class HRU_ORGANIC(HRU):
+        hru_id: int = 1
+        land_use_class: str = "FOREST"
+        veg_class: str = "FOREST"
+        soil_profile: str = "SOILP_ORG"
+        aquifer_profile: str = "[NONE]"
+        terrain_class: str = "[NONE]"
+
+    @dataclass
+    class HRU_BEDROCK(HRU):
+        hru_id: int = 2
+        land_use_class: str = "FOREST"
+        veg_class: str = "FOREST"
+        soil_profile: str = "SOILP_BEDROCK"
+        aquifer_profile: str = "[NONE]"
+        terrain_class: str = "[NONE]"
+
+    def __init__(self, *args, **kwds):
+        super().__init__(*args, **kwds)
+
+        self.config.update(
+            identifier="canadianshield",
+            # hrus=(HYPR.HRU(),),
```

Comment:
No HRU is initialized ?

Reply:
We're using the default one ?

Reply:
Yes that is an "oubli", can you confirm that the fix is ok please (not 100% sure).